lib/pymedphys/tests/dicom/test_anonymise.py

```diff
@@ -458,6 +459,11 @@ def test_anonymise_cli(tmp_path):
 
 def _test_anonymise_cli_for_file(tmp_path, test_file_path):
     temp_filepath = str(tmp_path / "test.dcm")
+
+    python_executable = pmp_test_utils.get_executable_even_when_embedded()
+
+    dicom_anonymise_cli = [python_executable, "-m", "pymedphys", "dicom", "anonymise"]
```

Comment:
dicom_anonymise_cli
inconsistent with 
def get_pymedphys_dicom_cli():

approach
DRY, but if you are going to, at least do so consistently.

suggest lifting 
def get_pymedphys_dicom_cli():

to a shared module

Reply:
The idea here was to try and touch the tests as little as possible while making it so that `pymedphys` could be called with `python -m pymedphys` instead.

...need to do that in order to be able to get `pymedphys.bat dev tests` to run.

This was just a way to propagate that change throughout the tests here. Personally, there's a huge amount I'd do to this file, the biggest of which is actually using python's tempfile logic. For now, I'd rather leave this for what it was (putting out a fire), as opposed to a clean up. For a clean up here is a lot of work.

Reply:
OK.
How about creating one or more issues and reference the files (tests) where this technical debt is being accumulated.
We can go through the issues later and notify the owners of the tests.
It would be best if the centralized function was created first, and then the owners notified that they can make use of those to retire the technical debt.
But the owners could turn around and say "I didn't add that code, why am I being asked to refactor it".

So the minimum is to just point out in an issue that this is something worth addressing at some point.
And clean up can be done a bit at a time.  Otherwise you end up with large PRs.  A single file being extensively modified isn't too bad.  But a single file having a few PRs where each PR addresses a particular cleanup is not so bad.

Reply:
I partly missed the tree for the forest here in your comment. I think in the end I was able to address the heart of what you were asking. Not exactly sure what to write in an issue for the rest of this though.